### PR TITLE
FOCUS 1.1 changelog updates

### PR DESCRIPTION
**Pull request by flanakin**
Changelog updates for FOCUS 1.1

### CODE REVIEW

**Comment by jpradocueva** [comment]
### Action Items from Task Force 2 meeting on September 25:
- [ ] [TF2-[#548](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/548)] Michael, @flanakin : Review metadata updates and ensure they are captured correctly in the changelog.
- [ ] [TF2-[#548](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/548)] Michael:, @flanakin  Verify that all recent changes, especially breaking ones related to commitment and utilization columns, are properly reflected in the changelog.
- [ ] [TF2-[#548](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/548)] Team: Agree on standardized changelog formatting and implement tools to automate consistent formatting across PRs.

**Comment by ijurica** [comment]
I've just noticed that we are not keeping track of appendix introductions. Not sure if we should. Just saying

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,13 +5,60 @@ All notable changes to this project will be documented in this file.
...
+
+- `CommitmentDiscountId` column updates:
+  - Must be globally unique within the provider.
+  - Should be a fully-qualified identifier.
+- `ConsumedQuantity` column updates:
+  - Must be null when `CommitmentDiscountStatus` is "Unused".
+- `ConsumedUnit` column updates:
+  - Must be null when `ChargeClass` is not "Correction" and `ChargeCategory` is not "Usage".
+  - Must be null when `ChargeClass` is not "Correction" and `ChargeCategory` is "Usage" and `CommitmentDiscountStatus` is "Unused".
+  - May be null when `ChargeCategory` is "Usage" and `ChargeClass` is "Correction.
```
Comment (cnharris10) [inline CHANGELOG.md:40]:
```suggestion
  - May be null when `ChargeCategory` is "Usage" and `ChargeClass` is "Correction".
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,13 +5,60 @@ All notable changes to this project will be documented in this file.
...
+- `EffectiveCost` column updates:
+  - When `CommitmentDiscountStatus` is "Unused", must be the difference between the used commitment amount and the portion of the total commitment purchase applicable for the charge period.
```
Comment (cnharris10) [inline CHANGELOG.md:42]:
```suggestion
  - When `CommitmentDiscountStatus` is "Unused", must be the difference between the used commitment discount amount and the portion of the total commitment discount purchase applicable for the charge period.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,13 +5,60 @@ All notable changes to this project will be documented in this file.
...
 
-[All unreleased changes](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/compare/1.0-cr...working_draft)
+[All unreleased changes](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/compare/1.1...working_draft)
 
 <br>
 -->
 
+## v1.1
+
+<sup>Announced November 12, 2024</sup>
```
Comment (cnharris10) [inline CHANGELOG.md:17]: Nothin' ... sup with you?

**Review by ahullah — APPROVED** [review]

**Review by cnharris10 — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (jpradocueva) [inline CHANGELOG.md:17]:
```suggestion
<sup>Announced November 2024</sup>
```

**Comment by jpradocueva** [comment]
Approved on Members call on Sep 26.

**Review by udam-f2 — APPROVED** [review]

**Comment by jpradocueva** [comment]
### Action Items from Members' call on Sep 26:
- [ ] [TF1-[#548](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/548) ] Members to double-check their PRs for any missed updates in the change log.

**Pull request merged**